lib/runPrettier.js

```diff
@@ -0,0 +1,70 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+/*
+ * Prettier must be ran from
+ * command line to make use of command line output.
+ * Use the bin file identified by Prettier's package.json
+ */
+function getPrettierBinLocation() {
```

Comment:
Any reason your not using arrow functions here? It doesn't really matter but the rest of the project does and consistency is good. The whole point of this PR yeah? Haha